jobs/JGLOBAL_MARINE_ANALYSIS_FINALIZE

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+source "${HOMEgfs}/ush/preamble.sh"
+export WIPE_DATA="NO"
+export DATAjob="${DATAROOT}/${RUN}marineanalysis.${PDY:-}${cyc}"
```

Comment:
Is this the RUNDIR directory where the files that persist across tasks are kept? If so, should it maybe be in the marineanl config so that it can be accessed by other tasks?

Reply:
It was supposed to look like this
![rundir](https://github.com/user-attachments/assets/23d20c12-c73f-406d-a577-956c6c868b97)

which it does for the most part except for the addition of `PDY`. I don't know if this is acceptable. It seems that `DATA` is hard-coded in the other jobs that share 1 `DATA` directory.

Reply:
@AndrewEichmann-NOAA , the pink boxes above survive through the cycle and are accessible for other tasks. I'm not exactly sure what you want me to add to the config, ```export DATAjob="${DATAROOT}/${RUN}marineanalysis.${PDY:-}${cyc}" ```?

Reply:
I believe `$DATA` is created before the configs are sourced, so probably not going to work